docs/semgrep-app/findings.md

```diff
@@ -205,14 +205,16 @@ To **fix a finding**:
 
 <RemoveRule />
 
-#### Viewing and adding comments to finding's history
+#### Viewing and adding comments to finding's activity
 
-To **view and add comments** to history of a finding:
+To **view and add comments** to activity history of a finding:
 
-1. Click the <i className="fa-regular fa-arrow-up-right-from-square inline_svg"></i> **Finding details** icon next to the finding’s status.
-2. Find the comments in the **History** section.
+1. Click the heading of the finding card.
+2. Find the comments in the **Activity** section.
```

Comment:
@sabrinabrogren where can I find the new plus icon in the circles? I cannot find it in FontAwesome. Please, if you have a link to this icon and we can reuse it in the docs, let us know. :-) <img width="41" alt="Screenshot 2023-01-31 at 21 58 12" src="https://user-images.githubusercontent.com/26669307/215820825-69177541-f8a9-4fad-bbb6-a583ca75734a.png">

Reply:
Ah, yeah that's actually combination of an icon and the [mantine timeline component](https://mantine.dev/core/timeline/). So the blue circle is from the component, and the plus in the middle is the actual icon. Should I still add the icon to the docs or would that be confusing?

Reply:
Please, if we can somehow add this icon, it will be great. @sabrinabrogren